vgen/src/main/resources/io/gapi/vgen/php/php_discovery_gapic.yaml

```diff
@@ -0,0 +1,4 @@
+type: io.gapi.vgen.ConfigProto
```

Comment:
Right now this naming pattern is consistent with Ruby and Java. If possible, I would prefer to leave it as-is for now and address this for all languages in a separate PR.